sources/Cargo.lock

```diff
@@ -2014,20 +2014,6 @@ dependencies = [
  "migration-helpers",
 ]
 
-[[package]]
```

Comment:
These were added in the second commit of the series, and then removed here. Please clean up so that they only appear in the commit where they were added.